apps/api/src/domain/models/digital-text/commands/add-photograph-to-digital-text-page/photograph-added-to-digital-text-page.event-handler.ts

```diff
@@ -0,0 +1,29 @@
+import { Inject } from '@nestjs/common';
+import { CoscradEventConsumer, ICoscradEventHandler } from '../../../../common';
+import {
+    DIGITAL_TEXT_QUERY_REPOSITORY_PROVIDER_TOKEN,
+    IDigitalTextQueryRepository,
+} from '../../queries/digital-text-query-repository.interface';
+import { PhotographAddedToDigitalTextPage } from './photograph-added-to-digital-text-page.event';
+
+@CoscradEventConsumer('PHOTOGRAPH_ADDED_TO_DIGITAL_TEXT_PAGE')
+export class PhotographAddedToDigitalTextPageEventHandler implements ICoscradEventHandler {
```

Comment:
You need to register the event and event handler in the `photograph` module. Otherwise, the event won't actually be handled at the `e2e` level.